### ISSUE TITLE
Links to waypoints break after loading media from the place manu to the scene

### ISSUE THREAD

**Issue by mauriciomartinez-itesm** [mozilla/hubs#4003]
**Description**
When trying to use a waypoint, the scene refreshes if you've placed a 3D object, Gif, Avatar or Scene, from the menu (even if you delete it afterwards), this is a problem for activities with breakout rooms.

**To Reproduce**
1. Go to this scene: https://hubs.mozilla.com/py5vGfo/pruebas
2. Click on Place
3. Select 3D Object, Gif, Avatar or Scene
4. Try to enter a breakout room
5. The page will be refreshed instead of transporting to the waypoint

**Expected behavior**
Place anything and be able to go to the waypoint

**Hardware** 
 - Device: Desktop, HMD (Quest v1)
 - OS: Windows, Quest Android / Oculus Browser
 - Browser: Chrome / Oculus Browser

**Additional context**
Apparently it doesn't happen with the hubs school v1.0 IDK why it just happens here

**Comment by mauriciomartinez-itesm** [comment]
After testing a bit and working with Christian V. on Discord, we've found out that this problem is happening because the link does not have an "/" at the end of the URL, once you put an / at the end it works as intended. I'm going to leave this open so it's still something visible for the Mozilla Team.

TL;DR:

hubs.mozilla.com/roomcode/roomname doesn't work while hubs.mozilla.com/roomcode/roomname/ does

**Comment by Exairnous** [comment]
#4272 has been merged and I can confirm that this has been fixed in both Chromium 91.0.4472.77 and Firefox 89.0 in Linux.
